src/main/java/org/primeframework/mvc/parameter/el/MemberAccessor.java

```diff
@@ -204,6 +205,10 @@ private Map<String, Field> findFields() {
     for (Map.Entry<String, Field> entry : ReflectionUtils.findFields(this.currentClass).entrySet()) {
       if (ReflectionUtils.areAnyAnnotationsPresent(entry.getValue(), unWrappedAnnotations)) {
         fields.putAll(ReflectionUtils.findFields(entry.getValue().getType()));
+      } else if (entry.getValue().isAnnotationPresent(NamedParameter.class)) {
```

Comment:
Do we need this code if `ReflectionUtils` knows how to recognize our new methods and properties?

Reply:
At the moment ReflectionUtils will return the NamedParameter methods in the property map, but they will not show up if you use `findFields` or `findMethods`. My reasoning was that `PropertyInfo` is something we created in ReflectionUtils and not part of `java.lang`. Since NamedParameter is also something we created I thought it was okay to include that there, but adding in the synthetic named parameters might not be what you would expect just looking for members.

So since `findFields` doesn't return an entry for the annotated name need to look it up here.

Happy to include the NamedParameter keys in the maps for fields and methods if you think that makes more sense.